source/elements/oneDAL/source/notations.rst

```diff
@@ -4,4 +4,57 @@
 Mathematical Notations
 ======================
 
+.. list-table::
+   :widths: 27 73
+   :header-rows: 1
+
+   * - Notation
+     - Definition
+
+   * - :math:`N`
+     - Number of feature vectors in a :term:`dataset <Dataset>`.
+
+   * - :math:`p`
+     - Number of features in a :term:`dataset <Dataset>`.
+
+   * - :math:`x_i` or :math:`x'_i`
```

Comment:
the description of x,x', (they are described together) and y,y' (they are described separately) appears to be inconsistent